Bio/PDB/parse_pdb_header.py

```diff
@@ -132,6 +132,52 @@ def parse_pdb_header(infile):
     return _parse_pdb_header_list(header)
 
 
+def _parse_remark_465(line):
+    """Parse missing residue remarks.
+
+    Returns a dictionary describing the missing residue.
+    The specification for REMARK 465 at
+    http://www.wwpdb.org/documentation/file-format-content/format33/remarks2.html#REMARK%20465
+    only gives templates, but does not say they have to be followed.
+    So we assume that not all pdb-files with a REMARK 465 can be understood.
+
+    Returns a dictionary with the following keys:
+    "model", "res_name", "chain", "ssseq", "insertion"
+    """
+    if line:
+        # Note that line has been stripped.
+        assert line[0] != " " and line[-1] not in "\n ", "line has to be stripped"
+    pattern = (r"""
```

Comment:
When using `re.VERBOSE` I prefer to do `pattern = re.compile(r"raw pattern", re.VERBOSE)`, then `match = pattern.match(line)` - this prevents trying to use the pattern again and forgetting the verbose flag.